A list of repository events are throwing java.lang.IllegalStateException

Hi,

for the following events:

 void transferred(@Repository.Transferred GHEventPayload.Repository repository)
 void renamed(@Repository.Renamed GHEventPayload.Repository repository)
 void unarchived(@Repository.Unarchived GHEventPayload.Repository repository)
 void privatized(@Repository.Privatized GHEventPayload.Repository repository)
 void publicized(@Repository.Publicized GHEventPayload.Repository repository)

there is the following exception
ex: 
`Caused by: java.lang.IllegalStateException: Parameter subscribing to a GitHub 'release' event must be of type 'org.kohsuke.github.GHEventPayload$Release' or 'io.quarkiverse.githubapp.GitHubEvent'. Offending method: com.gitrealm.bot.RepositoryListener#void renamed(org.kohsuke.github.GHEventPayload$Repository repository)`

It looks like the mapping is done with the ` @Release(Renamed.NAME)` instead of `@Repository(Renamed.NAME)`

--- a/events/src/main/java/io/quarkiverse/githubapp/event/Repository.java
+++ b/events/src/main/java/io/quarkiverse/githubapp/event/Repository.java
@@ -55,7 +55,7 @@ public @interface Repository {
         String NAME = Actions.EDITED;
     }
 
-    @Release(Privatized.NAME)
+    @Repository(Privatized.NAME)
     @Target(PARAMETER)
     @Retention(RUNTIME)
     @Qualifier
@@ -64,7 +64,7 @@ public @interface Repository {
         String NAME = Actions.PRIVATIZED;
     }
 
-    @Release(Publicized.NAME)
+    @Repository(Publicized.NAME)
     @Target(PARAMETER)
     @Retention(RUNTIME)
     @Qualifier
@@ -73,7 +73,7 @@ public @interface Repository {
         String NAME = Actions.PUBLICIZED;
     }
 
-    @Release(Renamed.NAME)
+    @Repository(Renamed.NAME)
     @Target(PARAMETER)
     @Retention(RUNTIME)
     @Qualifier
@@ -82,7 +82,7 @@ public @interface Repository {
         String NAME = Actions.RENAMED;
     }
 
-    @Release(Transferred.NAME)
+    @Repository(Transferred.NAME)
     @Target(PARAMETER)
     @Retention(RUNTIME)
     @Qualifier
@@ -91,7 +91,7 @@ public @interface Repository {
         String NAME = Actions.TRANSFERRED;
     }
 
-    @Release(Unarchived.NAME)
+    @Repository(Unarchived.NAME)
     @Target(PARAMETER)
     @Retention(RUNTIME)
     @Qualifier

